### DISCUSSION TITLE
python code is recognized as html ccold when commenting it out

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#11584] [Q&A]
### Description

As shown in the video below, I don't why python code in a quarto code block is regarded as html when commenting them out. 

Has anybody encountered similar issue before? 

https://github.com/user-attachments/assets/424c6ca6-ddda-4d60-925c-44209ed96836

**Comment by mcanouil** [comment]
Because that's the default and imperfect comment syntax for markdown.
In the end, it's up to your editor and independently of Quarto.

There are many threads here and on the internet about this.

- https://github.com/quarto-dev/quarto-cli/discussions/3330
